caom2-search-server/CHANGELOG.md

```diff
@@ -26,8 +26,15 @@
   * This is driven by the CADC's MAQ (Multi Archive Query) project to provide access to external data sources.
 * Upgraded support for the browser tests to Selenium 3.
 
-## September 12, 2019
+## September 12th, 2019
 
 ### 2.11.0
 * Major change: remove support for MAQ toggle
 * Change connection pool creation and configuration
+
```

Comment:
All recent activity is at the top (sorted most recent to least recent).  I'm not sure how the 2019 entry got down there.

Reply:
updated.